app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -28,30 +30,42 @@ IUSE=""
 
 DEPEND=">=dev-lang/go-1.3
 	app-arch/cpio
-	sys-fs/squashfs-tools"
+	sys-fs/squashfs-tools
+	dev-perl/Capture-Tiny"
 RDEPEND="!app-emulation/rocket"
 
-src_unpack() {
-	local cache="${S}/stage1/rootfs/usr_from_coreos/cache"
+src_configure() {
+	local myeconfargs=(
+		--with-stage1=host
```

Comment:
So there are two possibilities:
- Use `--with-stage1=host` and the build will not need to download an older version of the CoreOS image from internet. Instead, it will pick at run-time systemd and other binaries from the host. This option was added for Linux distributions such as Fedora and Debian because they forbid internet access during the build process, and forbid reusing pre-compiled binaries from third-parties (CoreOS) instead of compiling from source.
- Use `--with-stage1=coreos` (as it was before and it is still the default I think) and it will attempt to download the CoreOS image. Since ebuild don't allow internet access during the build, it should be pre-downloaded in `rkt-9999.ebuild` but the URL must be fixed as commented below.

/cc @krnowak